sdk/daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/crypto/Hash.scala

```diff
@@ -50,6 +59,18 @@ object Hash {
         extends HashingError("Contract IDs are not supported in contract keys.")
   }
 
+  sealed abstract class NodeHashingError(val msg: String) extends Exception with NoStackTrace
```

Comment:
Why do you need to suppress the stack trace? This might actually be helpful for debugging.